recipes/r-basetheme/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = '0.1.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-basetheme
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/basetheme_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/basetheme/basetheme_{{ version }}.tar.gz
+  sha256: 2b378aaa9b8b0b6a66357b42eac7fcdb5dc09c7dbd243e8f3680347c18269379
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('basetheme')"           # [not win]
+    - "\"%R%\" -e \"library('basetheme')\""  # [win]
+
+about:
+  home: https://github.com/KKPMW/basetheme
+  license: GPL-2.0-or-later
```

Comment:
```suggestion
  license: GPL-2.0-only
```

Reply:
@cbrueffer Good catch! This is a good reminder that the skeleton helper needs updating to be more nuanced for GPL-2 licenses:

https://github.com/bgruening/conda_r_skeleton_helper/pull/49
https://github.com/bgruening/conda_r_skeleton_helper/blob/9d4d7f069322bf272893829106693d819aa7fac2/run.py#L112